projects/core/src/organization/facade/permission.service.spec.ts

```diff
@@ -182,4 +184,21 @@ describe('PermissionService', () => {
       );
     });
   });
+
+  describe('get permission types', () => {
+    it('getTypes() should trigger load permission types when they are not present in the store', () => {
```

Comment:
need to check also positive scenario
![image](https://user-images.githubusercontent.com/8040837/81001553-32d79480-8e48-11ea-8d4f-1e1664b48210.png)